subprojects/docs/src/docs/userguide/building_java_projects.adoc

```diff
@@ -242,7 +231,7 @@ To use Java 6 or Java 7, the following tasks need to be configured:
 
 The following sample shows how the `build.gradle` needs to be adjusted. In order to be able to make the build machine-independent, the location of the old Java home and target version should be configured in `GRADLE_USER_HOME/gradle.properties` footnote:[For more details on `gradle.properties` see <<build_environment.adoc#sec:gradle_configuration_properties,Gradle configuration properties>> ] in the user's home directory on each developer machine, as shown in the example.
 
-==== Example: Configure Java 6 build
+==== Example: Configure Java 7 build
 
 [source,properties]
 .gradle.properties
```

Comment:
I think we missed a sample here... unless there is a restriction cross-compiling Java 6/7 using the Kotlin DSL.

Reply:
Yes. See https://github.com/gradle/gradle/issues/6442#issuecomment-416021234. Since these chapters use snippets from many samples, that also include other snippets used by other chapters, it's hard to translate a whole chapter without also translating pieces of other chapters. And doing that would lead to huge PRs.